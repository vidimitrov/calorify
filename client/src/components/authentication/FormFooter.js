import styled from 'styled-components';

const FormFooter = styled.div`
  background-color: #f6f6f6;
  border-top: 1px solid #e5c769;
  padding: 25px;
  text-align: center;
  -webkit-border-radius: 0 0 10px 10px;
  border-radius: 0 0 10px 10px;
`;

export default FormFooter;
