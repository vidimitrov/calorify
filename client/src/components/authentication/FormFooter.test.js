import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FormFooter from './FormFooter';

it('Authentication/FormFooter component renders correctly', () => {
  const tree = renderer.create(<FormFooter>Some footer</FormFooter>).toJSON();
  expect(tree).toMatchSnapshot();
});
