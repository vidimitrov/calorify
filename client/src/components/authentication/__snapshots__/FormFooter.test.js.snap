// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/FormFooter component renders correctly 1`] = `
.c0 {
  background-color: #f6f6f6;
  border-top: 1px solid #e5c769;
  padding: 25px;
  text-align: center;
  -webkit-border-radius: 0 0 10px 10px;
  border-radius: 0 0 10px 10px;
}

<div
  className="c0"
>
  Some footer
</div>
`;
